tests/lct_test.py

```diff
@@ -95,23 +93,19 @@ def test_chirp_multiply():
     dus, all_fvals, all_uvals = _vals()
     sigs = list(zip(dus, all_fvals))
     r = _cast_from_complex_signal(sigs, lct.chirp_multiply, _Q_CM)
-    _ndiff_files(
-        data_dir.join("cm_signals.txt"),
-        pkio.write_text(work_dir.join("cm_signals_actual.txt"), r),
-        work_dir.join("ndiff.out"),
-        data_dir,
+    pkunit.file_eq(
+        expect_path=data_dir.join("cm_signals.ndiff"),
```

Comment:
I do wonder if this couldn't be done with case_dirs. There seems to be a lot of copy and paste here.